### PR TITLE
[i18n] New, Add italian translation

### PR DESCRIPTION
**Pull request by nbenaglia**
Add italian translation

### CODE REVIEW

**Comment by dpgaspar** [comment]
Hi @nbenaglia,

Awesome! can you please add it to docs also?
here: https://github.com/dpgaspar/Flask-AppBuilder/blob/master/docs/i18n.rst

**Comment by nbenaglia** [comment]
> Hi @nbenaglia,
> 
> Awesome! can you please add it to docs also?

Done!

File: flask_appbuilder/translations/it/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -0,0 +1,766 @@
...
+msgstr "Relazione"
+
+#: flask_appbuilder/models/mongoengine/filters.py:101
+#: flask_appbuilder/models/sqla/filters.py:193
+msgid "Relation as Many"
+msgstr "Relazione di Many"
+
+#: flask_appbuilder/models/sqla/filters.py:97
+msgid "Ends with"
+msgstr ""
```
Comment (dpgaspar) [inline flask_appbuilder/translations/it/LC_MESSAGES/messages.po]: "Termina con" ?

Comment (nbenaglia) [reply flask_appbuilder/translations/it/LC_MESSAGES/messages.po]: Yes, fixed now.

**Pull request merged**